test_apps/test_app/test/ens_spec.js

```diff
@@ -19,7 +19,8 @@ config({
         `ENSRegistry.methods.setOwner('${rootNode}', web3.eth.defaultAccount).send().then(() => {
           ENSRegistry.methods.setResolver('${rootNode}', "$Resolver").send();
           Resolver.methods.setAddr('${rootNode}', '${address}').send();
-        });`
+        });
+        null;`
```

Comment:
What does this do?

Reply:
this code is run via the code runner.
When we return a promise we resolve/catch it.
however the promise never gets resolve/catch. (ws)
Adding null ensure that what is returned is not a promise